locksmith/src/app.ts

```diff
@@ -1,6 +1,17 @@
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import express from 'express'
+import tracing from '@opencensus/nodejs'
+import { JaegerTraceExporter } from '@opencensus/exporter-jaeger'
+
+const env = process.env.NODE_ENV || 'development'
+const config = require('../config/config')[env]
+
+const exporter = new JaegerTraceExporter(config.jaeger)
+
+if (env != 'test') {
```

Comment:
There maybe a slightly better place for this, but reason for being here remains the same. The tracing library intercepts the HTTP library in order to provide details on requests, timings, etc. 

Our test using the testing library `supertest` in order to provide a reasonable way  to assert on request being made during the test. 

To be expected, these together create unexpected results

Reply:
I guess there's no real downside to having this not run in the test environment, right?

Reply:
I don' think so, but it was a bit weird and unexpected to see this behavior